### ISSUE TITLE
Provide keyboard shortcut for Distill Page feature

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#8372]
## Description 
We currently offer a `Distill Page` feature (found under hamburger menu). This condenses the page content and removes the images. While we do offer this, there is no keyboard shortcut for this

If we implement the Speed Reader feature, we'd want to make sure there is a keyboard shortcut also

**Comment by bsclifton** [comment]
@simonhong would you be able to help me with this one?

**Comment by simonhong** [comment]
We can add our new accelerator via `chrome/browser/ui/views/accelerator_table.cc`.
Our shortcut for open tor window is already there.

**Comment by bsclifton** [comment]
Added `speedreader` label as I believe we want to make that user facing (and perhaps hide distill)
cc: @rebron

**Comment by NumDeP** [comment]
You know what would be good? [Ctrl+5] [:)](https://github.com/brave/brave-browser/issues/14704)

**Comment by jonsnow231** [comment]
Still waiting for this. Customizable hotkeys without an add-on would make Brave a lot better.

**Comment by valynor** [comment]
@bsclifton @simonhong 

It's 2024 and unfortunately there's still no keyboard shortcut to manually toggle Speedreader. 
Please consider adding this, I use Speedreader every day and this omission is a major annoyance. :-/

**Comment by valynor** [comment]
@vimfn 
We know about the auto speedreader function and that's not what we want - e.g. I don't need the speedreader on all sites, I need a way to start it quickly with a keyboard shortcut.

**Comment by NSurtsev** [comment]
Almost 5 years passed, still no shortcut.
What a shame!

**Comment by NunoSempere** [comment]
Also interested in this feature!

**Comment by TuralAsgar** [comment]
Where is shortcut?

**Comment by Salvora** [comment]
+1 for this and bump

**Comment by levontumanyan** [comment]
+1

**Comment by iubito** [comment]
Poor sighted, I use the speed reader equivalent in Firefox, because Brave doesn't have the F9 shortcut. Please, add it!!!
The speedreader is sometimes a hack to bypass the paywall on local newspaper, this is just a bonus ;)

**Comment by jyungtong** [comment]
while waiting for this feature, i'm using workaround by installing some random reader extension, then set shortcut for that reader extension.

**Comment by kago76** [comment]
+1

**Comment by localcatgirl-dev** [comment]
+1

**Comment by mevand** [comment]
I was LITERALLY using Edge just for this feature before I tried Brave. F9 was perfect!

+1

**Comment by zeke-jeske** [comment]
+1

As a simple workaround, try `<Ctrl+L> Tab Enter`. Basically, use your keyboard to focus the actual Speedreader button and activate it. Honestly, this hardly takes more time than pressing a single chord would.

I'm running Brave 146.1.88.134 on Ubuntu; the button placement might be different on different versions of Brave.

**Comment by mevand** [comment]
Thanks @zeke-jeske. Ctrl-L Tab Enter works the same in Windows 11 as does Command-L Tab Enter on macOS.

**Comment by valynor** [comment]
@bsclifton  It's been over 6 years now and this feature seems like it would be really trivial to add. Could you please consider adding this?

**Comment by BitSynKed** [comment]
Also hoping this gets added. In addition, the shortcut should be a toggle to turn speedreader both on _and_ off. This would make it great for people who use the auto speedreader setting so they can easily turn off speedreader on pages when they want to.
